COMMERCE-11957

victorcdp opened the pull request:
https://liferay.atlassian.net/browse/COMMERCE-11957

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

noornajj commented:
ci:test:sf

noornajj commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-3.liferay.com/job/test-portal-source-format/4302/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-3.liferay.com//userContent/jobs/test-portal-source-format/builds/4302/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 2 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1e6c9406ee9f3019dae917514b2ba8fc12b9819a">1e6c9406ee9f3019dae917514b2ba8fc12b9819a</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/noornajj/liferay-portal/tree/LPS-117316">LPS-117316</a><br/>Branch GIT ID: <a href="https://github.com/noornajj/liferay-portal/commit/3197dae60c776778ced561db32fa960918107d32">3197dae60c776778ced561db32fa960918107d32</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-source-format/4302/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-source-format/builds/4302/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-3.liferay.com/job/test-portal-source-format/4302/">test-portal-source-format#4302</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-source-format/builds/4302/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> sf<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/100">brianikim#100</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/12/Release/1039.aspx">/Liferay DXP 7.3/Pull Request/ci:test:sf</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/12/Build/20356.aspx">master - noornajj > brianikim - PR#100 - 2020-07-24[14:32:04]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-20.liferay.com/job/publish-spira-report/1419/">publish-spira-report#1419</a><br />

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/6788/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-14.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6788/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 20 out of 20 jobs passed</h3><h3>:x: ci:test:relevant - 54 out of 57 jobs passed in 1 hour 57 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d5618c8095cf21c8ba5eec5b944710562005ff4a">d5618c8095cf21c8ba5eec5b944710562005ff4a</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/2aa558d7587a046d654b60e91d9b21cb5e106df3">2aa558d7587a046d654b60e91d9b21cb5e106df3</a></p><details><summary><strong>ci:test:stable - 20 out of 20 jobs PASSED</strong></summary><details><summary><strong>20 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/420963/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305351/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/336986/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/443930/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/443931/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/419146/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305352/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/336816/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/420967/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/384604/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/415924/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/383441/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/421553/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305350/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/443929/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/421552/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376883/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/419143/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/419145/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/414024/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 54 out of 57 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/6788/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/336987/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</strong></strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/443932/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</strong></strike></a></li></ul></div><details><summary><strong>54 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/383440/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289240/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/420963/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/173634/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/384603/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289239/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/10</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/411661/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/11</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/393067/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/12</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/173633/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/13</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/420966/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/383439/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305349/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289238/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/393066/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/412849/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/336985/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305348/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/9</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/173632/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/173631/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/412850/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/10</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/173635/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/11</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/420965/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/412847/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/419144/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/336984/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/412846/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/393065/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/420964/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/384601/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/9</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305351/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/336986/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/443930/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/443931/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/419146/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305352/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/336816/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/420967/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/384604/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/415924/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/383441/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/421553/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305350/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/384602/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/393068/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/null</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/411662/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/443929/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/432483/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/414023/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/421552/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376883/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/419143/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/419145/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/412848/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/414024/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable</a></li></ul></details></details><h5>For more details click <a href="https://test-1-14.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6788/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/443932/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/443932//consoleText">AXIS_VARIABLE=0,label_exp=!master #443932</a><div><p>Please fix <strong>semantic versioning</strong> on <strong><a href="https://github.com/noornajj/liferay-portal/tree/LPS-117316">noornajj/LPS-117316</a></strong><pre><code>     [exec] 			+   return     boolean
     [exec] 		+   method     setSeparator(boolean)
     [exec] 			+   return     void
     [exec] 	&lt;   class      com.liferay.portal.kernel.servlet.taglib.ui.FormNavigatorEntryUtil
     [exec] 		+   method     filterVisibleFormNavigatorEntries(java.util.List,com.liferay.portal.kernel.theme.ThemeDisplay,java.lang.Object)
     [exec] 			+   access     protected
     [exec] 			+   access     static
     [exec] 			+   return     java.util.List
     [exec] 		+   method     getFormNavigatorEntries(java.lang.String,java.lang.String,com.liferay.portal.kernel.theme.ThemeDisplay,java.lang.Object)
     [exec] 			+   access     static
     [exec] 			+   return     java.util.List
     [exec] 	&lt;   class      com.liferay.portal.kernel.servlet.taglib.ui.JavaScriptMenuItem
     [exec] 		+   method     hasSeparator()
     [exec] 			+   return     boolean
     [exec] 		+   method     setSeparator(boolean)
     [exec] 			+   return     void
     [exec] 	&lt;   class      com.liferay.portal.kernel.servlet.taglib.ui.MenuItem
     [exec] 		+   method     hasSeparator()
     [exec] 			+   return     boolean
     [exec] 		+   method     setSeparator(boolean)
     [exec] 			+   return     void
     [exec] 	&lt;   class      com.liferay.portal.kernel.servlet.taglib.ui.URLMenuItem
     [exec] 		+   method     hasSeparator()
     [exec] 			+   return     boolean
     [exec] 		+   method     setSeparator(boolean)
     [exec] 			+   return     void
     [exec] 	&lt;   class      com.liferay.portal.kernel.servlet.taglib.ui.URLToolbarItem
     [exec] 		+   method     getData()
     [exec] 			+   return     java.util.Map
     [exec] 		+   method     setData(java.util.Map)
     [exec] 			+   return     void
     [exec] 	&gt;   interface  com.liferay.portal.kernel.servlet.taglib.ui.FormNavigatorEntry
     [exec] 		+   method     isVisible(com.liferay.portal.kernel.theme.ThemeDisplay,java.lang.Object)
     [exec] 			+   access     abstract
     [exec] 			+   return     boolean
     [exec] 	-   version    6.4.0
     [exec] 	+   version    6.6.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/portal-kernel/portal-kernel.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.kernel/3.99.0/7537d8ae969988eece30dc6000f28ab072e2770d/com.liferay.portal.kernel-3.99.0.jar</code></pre></p></div></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at d0e51c547f753dda645d703007c228dff7edd49f:</strong></summary><ol><li><div><h5><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/336987/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></h5><div><h6>Job Results:</h6><p>1476 Tests Passed.<br/>182 Tests Failed.</p></div><ol><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/336987//consoleText">AXIS_VARIABLE=0,label_exp=!master #336987</a><ol><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/336987//testReport/com.liferay.headless.delivery.resource.v1_0.test/BlogPostingImageResourceTest/testGetSiteBlogPostingImagesPage">BlogPostingImageResourceTest.testGetSiteBlogPostingImagesPage</a><pre><code>java.lang.IllegalArgumentException: Unsupported field name actions
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:176)
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:138)
	at com.liferay.headless.delivery.client.json.BaseJSONParser.parseToDTO(BaseJSONParser.java:90)
	at com.liferay.headless.delivery.client.problem.Problem.toDTO(Problem.java:33)
	at com.liferay.headless.delivery.client.resource.v1_0.BlogPostingImageResource$BlogPostingImageResourceImpl.getSiteBlogPostingImagesPage(BlogPostingImageResource.java:360)
	at com.liferay.headless.delivery.resource.v1_0.test.BaseBlogPostingImageResourceTestCase.testGetSiteBlogPostingImagesPage(BaseBlogPostingImageResourceTestCase.java:355)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$2.evaluate(TestExecutorRunnable.java:234)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.Ab...</code></pre></div></li><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/336987//testReport/com.liferay.headless.delivery.resource.v1_0.test/BlogPostingImageResourceTest/testGetSiteBlogPostingImagesPageWithFilterStringEquals">BlogPostingImageResourceTest.testGetSiteBlogPostingImagesPageWithFilterStringEquals</a><pre><code>java.lang.IllegalArgumentException: Unsupported field name actions
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:176)
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:138)
	at com.liferay.headless.delivery.client.json.BaseJSONParser.parseToDTO(BaseJSONParser.java:90)
	at com.liferay.headless.delivery.client.problem.Problem.toDTO(Problem.java:33)
	at com.liferay.headless.delivery.client.resource.v1_0.BlogPostingImageResource$BlogPostingImageResourceImpl.getSiteBlogPostingImagesPage(BlogPostingImageResource.java:360)
	at com.liferay.headless.delivery.resource.v1_0.test.BaseBlogPostingImageResourceTestCase.testGetSiteBlogPostingImagesPageWithFilterStringEquals(BaseBlogPostingImageResourceTestCase.java:462)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$2.evaluate(TestExecutorRunnable.java:234)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.port...</code></pre></div></li><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/336987//testReport/com.liferay.headless.delivery.resource.v1_0.test/BlogPostingImageResourceTest/testGetSiteBlogPostingImagesPageWithPagination">BlogPostingImageResourceTest.testGetSiteBlogPostingImagesPageWithPagination</a><pre><code>java.lang.IllegalArgumentException: Unsupported field name actions
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:176)
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:138)
	at com.liferay.headless.delivery.client.json.BaseJSONParser.parseToDTO(BaseJSONParser.java:90)
	at com.liferay.headless.delivery.client.problem.Problem.toDTO(Problem.java:33)
	at com.liferay.headless.delivery.client.resource.v1_0.BlogPostingImageResource$BlogPostingImageResourceImpl.getSiteBlogPostingImagesPage(BlogPostingImageResource.java:360)
	at com.liferay.headless.delivery.resource.v1_0.test.BaseBlogPostingImageResourceTestCase.testGetSiteBlogPostingImagesPageWithPagination(BaseBlogPostingImageResourceTestCase.java:492)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$2.evaluate(TestExecutorRunnable.java:234)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kerne...</code></pre></div></li><li>...</li></ol></div></li><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/336987//consoleText">AXIS_VARIABLE=1,label_exp=!master #336987</a><ol><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/336987//testReport/com.liferay.headless.delivery.resource.v1_0.test/MessageBoardAttachmentResourceTest/testGetMessageBoardMessageMessageBoardAttachmentsPage">MessageBoardAttachmentResourceTest.testGetMessageBoardMessageMessageBoardAttachmentsPage</a><pre><code>java.lang.IllegalArgumentException: Unsupported field name actions
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:176)
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:138)
	at com.liferay.headless.delivery.client.json.BaseJSONParser.parseToDTO(BaseJSONParser.java:90)
	at com.liferay.headless.delivery.client.problem.Problem.toDTO(Problem.java:33)
	at com.liferay.headless.delivery.client.resource.v1_0.MessageBoardAttachmentResource$MessageBoardAttachmentResourceImpl.getMessageBoardMessageMessageBoardAttachmentsPage(MessageBoardAttachmentResource.java:411)
	at com.liferay.headless.delivery.resource.v1_0.test.BaseMessageBoardAttachmentResourceTestCase.testGetMessageBoardMessageMessageBoardAttachmentsPage(BaseMessageBoardAttachmentResourceTestCase.java:362)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$2.evaluate(TestExecutorRunnable.java:234)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evalu...</code></pre></div></li><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/336987//testReport/com.liferay.headless.delivery.resource.v1_0.test/MessageBoardAttachmentResourceTest/testGetMessageBoardThreadMessageBoardAttachmentsPage">MessageBoardAttachmentResourceTest.testGetMessageBoardThreadMessageBoardAttachmentsPage</a><pre><code>java.lang.IllegalArgumentException: Unsupported field name actions
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:176)
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:138)
	at com.liferay.headless.delivery.client.json.BaseJSONParser.parseToDTO(BaseJSONParser.java:90)
	at com.liferay.headless.delivery.client.problem.Problem.toDTO(Problem.java:33)
	at com.liferay.headless.delivery.client.resource.v1_0.MessageBoardAttachmentResource$MessageBoardAttachmentResourceImpl.getMessageBoardThreadMessageBoardAttachmentsPage(MessageBoardAttachmentResource.java:629)
	at com.liferay.headless.delivery.resource.v1_0.test.BaseMessageBoardAttachmentResourceTestCase.testGetMessageBoardThreadMessageBoardAttachmentsPage(BaseMessageBoardAttachmentResourceTestCase.java:482)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$2.evaluate(TestExecutorRunnable.java:234)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluat...</code></pre></div></li><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/336987//testReport/com.liferay.headless.delivery.resource.v1_0.test/MessageBoardMessageResourceTest/testGetMessageBoardMessageMessageBoardMessagesPage">MessageBoardMessageResourceTest.testGetMessageBoardMessageMessageBoardMessagesPage</a><pre><code>java.lang.IllegalArgumentException: Unsupported field name actions
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:176)
	at com.liferay.headless.delivery.client.problem.Problem$ProblemJSONParser.setField(Problem.java:138)
	at com.liferay.headless.delivery.client.json.BaseJSONParser.parseToDTO(BaseJSONParser.java:90)
	at com.liferay.headless.delivery.client.problem.Problem.toDTO(Problem.java:33)
	at com.liferay.headless.delivery.client.resource.v1_0.MessageBoardMessageResource$MessageBoardMessageResourceImpl.getMessageBoardMessageMessageBoardMessagesPage(MessageBoardMessageResource.java:1086)
	at com.liferay.headless.delivery.resource.v1_0.test.BaseMessageBoardMessageResourceTestCase.testGetMessageBoardMessageMessageBoardMessagesPage(BaseMessageBoardMessageResourceTestCase.java:506)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$2.evaluate(TestExecutorRunnable.java:234)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule...</code></pre></div></li><li>...</li></ol></div></li><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/336987//consoleText">AXIS_VARIABLE=2,label_exp=!master #336987</a><ol><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/336987//testReport/com.liferay.journal.search.test/JournalArticleSearchTest/testOrderByDDMNumberFieldRepeatable">JournalArticleSearchTest.testOrderByDDMNumberFieldRepeatable</a><pre><code>org.junit.ComparisonFailure: expected:&lt;{3.14,20|12.34,16[.0]}&gt; but was:&lt;{3.14,20|12.34,16[]}&gt;
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.liferay.dynamic.data.mapping.test.util.search.TestOrderHelper.assertSearch(TestOrderHelper.java:232)
	at com.liferay.dynamic.data.mapping.test.util.search.TestOrderHelper.testOrderByDDMField(TestOrderHelper.java:356)
	at com.liferay.dynamic.data.mapping.test.util.search.TestOrderHelper.testOrderByDDMFieldRepeatable(TestOrderHelper.java:395)
	at com.liferay.dynamic.data.mapping.test.util.search.TestOrderHelper.testOrderByDDMNumberFieldRepeatable(TestOrderHelper.java:119)
	at com.liferay.journal.search.test.JournalArticleSearchTest.testOrderByDDMNumberFieldRepeatable(JournalArticleSearchTest.java:245)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$2.evaluate(TestExecutorRunnable.java:234)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.porta...</code></pre></div></li></ol></div></li></ol></div></li></ol></details></details></html>

brianikim commented:
Will finish reviewing this on Monday.

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/6788/">test-portal-acceptance-pullrequest(master)#6788</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-14.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6788/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> relevant<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/100">brianikim#100</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/12/Release/896.aspx">/Liferay DXP 7.3/Pull Request/ci:test:relevant</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/12/Build/20435.aspx">master - noornajj > brianikim - PR#100 - 2020-07-24[15:18:38]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-16.liferay.com/job/publish-spira-report/4506/">publish-spira-report#4506</a><br />

brianikim commented:
As an update, I am still reviewing this. will try to push the PR by tomorrow.

brianikim commented:
See https://github.com/SamZiemer/liferay-portal/pull/745

Commit:
COMMERCE-9815 - make the "default option" switch also able to be turned off

riccardo-alberti committed:
COMMERCE-8912 Check workflow status when searching for discounts

AlmirFe committed:
LPS-173302 Add a check to see if moduleName is null

georgel-pop-lr committed:
LPS-180479 Prevent aui:a to render shortcut icon if there is already an icon setup

Commit:
LPS-180479 Fix tests according to ticket changes.

Commit:
LPS-180479 SF

izaera committed:
LPS-186758 Add ESM support to Soy renderer

As we do in React renderer.

izaera committed:
LPS-186758 Migrate frontend-taglib-chart to ESM

Note that we have replaced the use of `TagResourceHandler` to output links
to the clay-charts CSS and SVG files by ESM imports that will be retrieved
as dependencies any time a chart is rendered.

julschong committed:
LPS-185768 Replace AMImageSizeProvider.getImageMaxSize() with DLFileEntryConfiguration.previewableProcessorMaxSize()
AMImageConfiguration.imageMaxSize property is deprecated and are to be removed next

julschong committed:
LPS-185768 Cleanup unused reference of AMImageSizeProvider

julschong committed:
LPS-185768 Remove AMImageSizeProvider and AMImageConfiguration.imageMaxSize(), no more usage
Lima team confirmed Okay to remove this configuration

julschong committed:
LPS-185768 Auto, gw baseline

georgel-pop-lr committed:
LPS-180479 Add source-formatter-suppressions.xml to ignore IllegalTaglibsCheck on portal-web/docroot/html/taglib/ui/icon/page.jsp

AlmirFe committed:
LPS-173302 Add debug log to see when moduleName is null

izaera committed:
LPS-186758 gradle baseline

julschong committed:
LPS-185768 Breaking Change

Commit:
LPS-184008 Update build-test and test.properties files to build u73 on jenkins

brittneyq committed:
LRCI-2735 Add validate PQL method

brittneyq committed:
LRCI-2735 Validate PQL when concating or adding PQL

julschong committed:
LPS-185768 Adding upgrade process to remove imageMaxSize from AMImageConfiguration

julschong committed:
LPS-185768 Auto, ant update-portal-osgi-configuration-properties

kildarep committed:
LPS-189219 Adjusting visibility of the popove body

Commit:
LPS-188749 save localization when a repeated field is created on a non default locale

nicolas-sss committed:
LPS-189252 Create UpgradeJavaPortletSharedSearchSettingsCheck

nicolas-sss committed:
LPS-189252 Generated

nicolas-sss committed:
LPS-189252 Create test for UpgradeJavaPortletSharedSearchSettingsCheck

nicolas-sss committed:
LPS-189252 SF

matthiasblaesing committed:
LPS-190095 Fix wrong URL usage in DeepL Translator

sergiojimcos committed:
LPS-189717 Add two validations: an API Schema can not have the same name in the same API Application and check that the API Application is valid

sergiojimcos committed:
LPS-189717 buildLanguage

sergiojimcos committed:
LPS-189717 Add test case to cover these validations

oliv-yu committed:
LPS-189258 portal-search-web: Wrap vocabulary tree styling around cadmin selector

oliv-yu committed:
LPS-189258 portal-search-web: Add Global Site into site list if it is missing

Commit:
LPS-180479 Fix gotoPreviewView macro

4lejandrito committed:
LPS-189717 Reusing the helper

Commit:
LPS-180479 Change variable in Workflow.gotoPreviewView macro

Brunno0liveira committed:
COMMERCE-9815 Add Test CanAssertDefaultOptionValuesAreSavedUsingToggleSwitch

oliv-yu committed:
LPS-189258 portal-search-web: Selected vocab IDs have trouble persisting (use stringified IDs, not numbers)

oliv-yu committed:
LPS-189258 portal-search-web: Fix some styling around vocabulary selection

Commit:
LPS-173756 SF, clean up the parameters of SF, these classes no longer contain method references

dantewang committed:
LPS-189793 Generate build.namespace as bundle symbolic name instead of portlet short name

dantewang committed:
LPS-189793 Delete redundant file. The file was added to override the build.namespace to avoid using "CT" for Change Tracking due to conflict with Content Targeting; now that we change name to bundle symbolic name, it's no longer needed.

4lejandrito committed:
LPS-189713 We only need to track the bundle if there are multicompany units

Commit:
LPS-189986 portal-impl - make WEB_SERVER_FORWARDED_PROTOCOL/HOST/PORT_ENABLED properties non-final to avoid flaky behavior caused by JIT runtime inlining

petershin committed:
LPS-189793 Set version

petershin committed:
LPS-189793 Regen Maven plugin descriptors

petershin committed:
LPS-189793 Regen

Jeremy-Ch committed:
LPS-187679 Update macro for viewing the filter details

petershin committed:
LPS-189793 Auto SF

Commit:
LPS-187768 Switch iText under AGPL (as dependency of JasperReport) for OpenPDF (an LGPL - licensed fork of iText)

murilostodolni committed:
LPS-189927 Create ObjectDBManagerUtil class to create index in object

murilostodolni committed:
LPS-189927 Use static method to create index

brittneyq committed:
LRCI-2735 Simplify

sergiojimcos committed:
LPS-189728 Change deletionType oto disassociate in relationships between API Schema and API Endpoint

sergiojimcos committed:
LPS-189728 Add validation if an user is setting an invalid API Schema in the APIs

sergiojimcos committed:
LPS-189728 buidLanguage

sergiojimcos committed:
LPS-189728 Add test case to check API Endpoint validation

ealonso committed:
LPS-190093 Adds support to editing info items on display pages

Jeremy-Ch committed:
LPS-187679 Add tests for the document filtering in Asset libraries

AliciaGarciaGarcia committed:
LPS-189427 add host to prevent NPE

Commit:
LPS-188867 Fix fileEntryUrl on workflow tasks

victorg1991 committed:
LPS-174149 Don't allow mapping boolean or categorization fields that cannot contain urls

kevenleone committed:
LPS-188219 Rename and Simplify

Commit:
LPS-189265 created Exception Mapper to ObjectFieldReadOnlyConditionExpressionException

tinatian committed:
LPS-189793 Auto generated

Commit:
LPS-190221 Add cached label to contributed fragments when defined as cacheabled

is-solutions-delivery committed:
LPS-190292 Customer Portal - New translations Language.json (Spanish)

is-solutions-delivery committed:
LPS-190292 Customer Portal - New translations Language.json (Japanese)

is-solutions-delivery committed:
LPS-190292 Customer Portal - New translations Language.json (Portuguese, Brazilian)

ricardocousolr committed:
LPS-190239 Remove heading from form success page

ricardocousolr committed:
LPS-190239 Fix test path

Commit:
LPS-187089 Add viewActivityNotPresent macro

Commit:
LPS-187089 Change removeBudgetValue click to deal with flaky cases

Commit:
LPS-187089 Improve editActivity and editMDF macros do deal with cases where the budget and activities can be removed

Commit:
LPS-187089 Add macros to validate budget and activity data on MDF Request Detail Page

Commit:
LPS-187089 Improve EDIT_ACTIVITY_BUTTON path to receive the activity name as parameter

Commit:
LPS-187089 Add paths to be used during activity and budget edit

Commit:
LPS-187089 Add paths to be used on MDF Request Detail Page validations

Commit:
LPS-187089 Add viewBudgetNotPresent macro

Commit:
LPS-182807 Add CanRemoveActivityWhileEditing test to verify that it is possible to remove an activity while editing

Commit:
LPS-187089 Update the old tests to deal with a change on editMDF macro

Commit:
LPS-189519 Fix EventDetails tests

carolmariaabb committed:
LPS-188867 Simplify fix by passing the ddmFormInstanceRecordId property when editing the form record in workflow tasks

ivicac committed:
LRAC-14252 - Set Content type to “application/zip” when returning exported data as a zip file

magjed4289 committed:
LRQA-81467 Test-fix

Commit:
LPS-190272 Field cannot be added after object is created

kevenleone committed:
LPS-190291 Testray Enhancements

brookedalton committed:
LPS-190297 Update path to be more specific

brookedalton committed:
Revert "LPS-189633 Find pagination when it is disabled"

This reverts commit ad031f5acf2ee8311b16b728a86f0fe62d3a7608.

matuzalemsteles committed:
LPS-190150 add reduced motion to the product menu

WYZ-Issac committed:
LPS-170177 Update the macro to view more action button

WYZ-Issac committed:
LPS-170177 Optimize test

jaydawu committed:
COMMERCE-8912 Add test CannotViewProductDiscountsWhenDiscountsAreSavedAsDraft

ling-alan-huang committed:
POSHI-556 Rename

ling-alan-huang committed:
LPS-189976 Auto SF

brianchandotcom committed:
LPS-190095 Sort

brianchandotcom committed:
LRCI-2735 prep next

Commit:
LPS-188295 Add case to map specific display page for web content article

brianchandotcom committed:
LPS-189927 SF

brianchandotcom committed:
object-service 1.0.160 prep next

Commit:
LPS-188295 Add case to map specific display page for blogs entry

Commit:
LPS-188295 Add case to map specific display page for document

Commit:
LPS-188295 Add case to map specific display page for object

Commit:
LPS-190268 Update JSONTBatchEngineExportTaskItemWriterImplTest to include multiCompany

Commit:
LPS-188295 Modify macro to select category

Commit:
LPS-188295 Add case to map specific display page for category

Commit:
LPS-188295 Modify macro to select site

Commit:
LPS-188295 Add case to map specific display page for commerce product

Jeremy-Ch committed:
LPS-190317 Update test to use a simple way to add multiple documents

Jeremy-Ch committed:
LPS-190318 Go to edit the document by UI

JoyceWang08211 committed:
LRQA-81454 Ignore test

Commit:
LPS-190321 Fix typo

Commit:
LPS-190323 Remove steps since component is enable by default

Commit:
LPS-190326 Sort

Commit:
LPS-190326 Modify macro to make it stable

Commit:
LPS-190326 Modify macro to return objectEntryId

Commit:
LPS-190326 Update usages

Commit:
LPS-190326 AutoSF

Commit:
LPS-190322 Add steps to wait for iframe loading

brianchandotcom committed:
LPS-189728 SF

brianchandotcom committed:
LPS-173302 Wordsmith

brianchandotcom committed:
LPS-189717 SF

brianchandotcom committed:
LPS-189717 Wordsmith

brianchandotcom committed:
LPS-189717 Regen

brianchandotcom committed:
LPS-189793 prep next

brianchandotcom committed:
LPS-189793 prep next

Commit:
COMMERCE-11957 remove periods and slashes when editing specification key

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

brianikim commented on modules/apps/commerce/commerce-product-options-web/src/main/resources/META-INF/resources/edit_cp_specification_option.jsp:
```diff
@@ -53,4 +53,8 @@ else {
+
+<liferay-frontend:component
+	module="js/edit_cp_option_and_value"
+/>
```
Why do we need this?

victorcdp replied:
This js updates the key and removes slashes or dots in real time if the user tries inputting one, it's just a frontend validation, it's in the Options page as well.

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7c8d932a1011e2edf2ab7e53d0472da80068ac98">7c8d932a1011e2edf2ab7e53d0472da80068ac98</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-11957">COMMERCE-11957</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/4da644ff88ed24758b04642f04768435747e9d7d">4da644ff88ed24758b04642f04768435747e9d7d</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-33.liferay.com/job/test-portal-source-format/2125/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/2125/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-33.liferay.com/job/test-portal-source-format/2125/">test-portal-source-format#2125</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/2125/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/100">brianikim#100</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2380447507">[master] ci:test:sf - victorcdp &gt; brianikim - PR#100 - 2023-07-12[09:04:05]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-33.liferay.com/job/publish-testray-report/8896/">publish-testray-report#8896</a><br/></div></div>

brianikim commented:
You can forward. Thanks.
